third_party/dotnet/devtools/src/generator/Templates/command.hbs

```diff
@@ -55,3 +55,10 @@ namespace {{rootNamespace}}.{{domain.Name}}
     {{/each}}
     }
 }
+
+namespace OpenQA.Selenium.DevTools.Json
+{
+    [global::System.Text.Json.Serialization.JsonSerializable(typeof(global::{{rootNamespace}}.{{domain.Name}}.{{className}}CommandSettings))]
+    [global::System.Text.Json.Serialization.JsonSerializable(typeof(global::{{rootNamespace}}.{{domain.Name}}.{{className}}CommandResponse))]
+    internal sealed partial class CdpSerializationContext;
```

Comment:
We have 2 options:
- Having only one single `CdpSerializationContext` for types from all cdp namespaces (v130, v131, v132) (and we know how to resolve collisions)
- Or separate json context per cdp version

I am not deep dived yet. Please suggest which option would be more friendly for us.